src/courseware/course/sequence/sequence-navigation/hooks.js

```diff
@@ -25,6 +30,9 @@ export function useSequenceNavigationMetadata(currentSequenceId, currentUnitId)
   const isLastSequence = sequenceIndex === sequenceIds.length - 1;
   const isLastUnitInSequence = unitIndex === sequence.unitIds.length - 1;
   const isLastUnit = isLastSequence && isLastUnitInSequence;
+  const sequenceNavigationDisabled = sequence?.navigationDisabled;
```

Comment:
```suggestion
  const sequenceNavigationDisabled = sequence.navigationDisabled;
```

It doesn't look like `sequence` is optional, at this point.